app/models/concerns/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
```

Comment:
- [ ] From the previous reviewer's screenshots, this file should be located in the `app/models` directory, not `app/models/concerns`.

Reply:
Thank you @DammyShittu 🙏🏻